statistics/testdata/integration_suite_out.json

```diff
@@ -7,19 +7,19 @@
         "└─IndexRangeScan_5 5.00 cop[tikv] table:exp_backoff, index:idx(a, b, c, d) range:[1,1], keep order:false"
       ],
       [
-        "IndexReader_10 3.00 root  index:Selection_9",
-        "└─Selection_9 3.00 cop[tikv]  eq(test.exp_backoff.b, 1)",
-        "  └─IndexFullScan_8 5.00 cop[tikv] table:exp_backoff, index:idx(a, b, c, d) keep order:false"
+        "TableReader_7 3.00 root  data:Selection_6",
+        "└─Selection_6 3.00 cop[tikv]  eq(test.exp_backoff.b, 1)",
+        "  └─TableFullScan_5 5.00 cop[tikv] table:exp_backoff keep order:false"
       ],
       [
-        "IndexReader_10 2.00 root  index:Selection_9",
-        "└─Selection_9 2.00 cop[tikv]  eq(test.exp_backoff.c, 1)",
-        "  └─IndexFullScan_8 5.00 cop[tikv] table:exp_backoff, index:idx(a, b, c, d) keep order:false"
+        "TableReader_7 2.00 root  data:Selection_6",
+        "└─Selection_6 2.00 cop[tikv]  eq(test.exp_backoff.c, 1)",
+        "  └─TableFullScan_5 5.00 cop[tikv] table:exp_backoff keep order:false"
       ],
       [
-        "IndexReader_10 3.00 root  index:Selection_9",
-        "└─Selection_9 3.00 cop[tikv]  ge(test.exp_backoff.d, 3), le(test.exp_backoff.d, 5)",
-        "  └─IndexFullScan_8 5.00 cop[tikv] table:exp_backoff, index:idx(a, b, c, d) keep order:false"
+        "TableReader_7 3.00 root  data:Selection_6",
+        "└─Selection_6 3.00 cop[tikv]  ge(test.exp_backoff.d, 3), le(test.exp_backoff.d, 5)",
+        "  └─TableFullScan_5 5.00 cop[tikv] table:exp_backoff keep order:false"
```

Comment:
Why are these needing to change?

Reply:
removed. If we push down partitionIds，cost will not change  and modifications above is unnecessary now.